example_template.md

```diff
@@ -36,12 +36,12 @@ These instructions explain how to build the example yourself and deploy it with
 For example:
```

Comment:
Why we have this in a different file than the README?

Reply:
I think this is just the template for all the examples.

Reply:
Yes! I see it now, never mind this